code/modules/surgery/organs/lungs.dm

```diff
@@ -334,12 +338,20 @@
 	// Healium
 		var/healium_pp = breath.get_breath_partial_pressure(breath_gases[/datum/gas/healium][MOLES])
 		if(healium_pp > gas_stimulation_min)
-			var/existing = H.reagents.get_reagent_amount(/datum/reagent/healium)
-			H.reagents.add_reagent(/datum/reagent/healium,max(0, 1 - existing))
-			H.adjustOxyLoss(-5)
-			H.adjustFireLoss(-7)
-			H.adjustToxLoss(-7)
-			H.adjustBruteLoss(-10)
+			if(prob(15))
+				to_chat(H, "<span class='alert'>Your head starts spinning and your lungs burn!</span>")
+				SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "chemical_euphoria", /datum/mood_event/chemical_euphoria)
+				H.emote("gasp")
+		else
+			SEND_SIGNAL(owner, COMSIG_CLEAR_MOOD_EVENT, "chemical_euphoria")
+		if(healium_pp > healium_para_min)
+			H.Unconscious(rand(30, 50))//not in seconds to have a much higher variation
+			if(healium_pp > healium_sleep_min)
+				H.Sleeping(min(H.AmountSleeping() + rand(100, 150), rand(200, 400)))
```

Comment:
if you're no longer using the healium reagent for this, it might be best to remove it (the healium reagent), so that botanists can't roll it as a strange seeds chem or something.

Reply:
mmm i could remove it true, unless there is a way to make the reagent sleep instead of knockdown

Reply:
reagents can do anything to the person a gas could do.

Reply:
Put this back into the reagent code.